pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -146,14 +146,26 @@ def test_date_range_businesshour(self):
 
     def test_range_misspecified(self):
         # GH #1095
+        with pytest.raises(ValueError):
```

Comment:
We know what the error message is here (you just added it).  Let's make this test (and all other relevant ones) stronger by checking the error message too.